### DISCUSSION TITLE
Unable to list server as public

### DISCUSSION THREAD

**Discussion by Paralda** [mbround18/valheim-docker#201] [Help!] [answered]
Decided to migrate my server from Windows to this, and for some reason, even though I can connect fine directly by IP, the server isn't showing up in the list of community servers.

I'm using default ports and can confirm they're all open. In addition, I get hung up on the "DungeonDB is starting" section of the log when starting the server, but I can generally still connect directly by IP even when that's happening.

Has anyone experienced this same problem, and if so, how did you fix it?

**Comment by CosmicHorrorDev** [comment, marked as answer]
There seems to be a lot of issues with getting peoples' servers to show up in the community listing. For reference I never see my server on the community list while some other people on my same server can (and this changes at random). Also this behavior is not specific to this image, I've seen people all over the internet with different configurations talk about being able to connect directly, but not seeing the server in the listing.

I've also never had issues directly connecting though so I would just always have that to fall back on.

Regarding the logs: the server start up is multi-threaded, so the messages get jumbled. It likely says game server connected up above the logs you were looking at.
